viaq-jenkins/fields.yml

```diff
@@ -0,0 +1,493 @@
+version: 2016.03.21.0
+
+defaults:
+  type: string
+
+
+_default_:
+  type: group
+  description: >
+    Contains common fields, corresponds to mappings of the _default_.
+    This mapping represents generic Jenkins configuration.
+  fields:
+    - name: "@timestamp"
+      type: date
+      format: yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ||yyyy-MM-dd'T'HH:mm:ssZ||dateOptionalTime
+      example: 2015-01-24T14:06:05.071Z
+      description: >
+        UTC value marking when the log payload was created, or when log payload was first collected if the creation time is not known; 
+        this is the log processing pipeline’s “best effort” determination of when the log payload was generated
+        FYI: the “@” prefix convention to note a field as being reserved for a particular use; in this case, most tools by default look for “@timestamp” with ElasticSearch
+        Collected by shipshift from Jenkins, when the message was emitted.
+      fields:
+        - name: raw
+          ignore_above: 256
+          type: string
+          index: not_analyzed
+
+    - name: geoip
+      description: >
+        geo-ip of the machine
+      type: object
+      object_struct:
+        dynamic: True
+        properties:
+          location:
+            type: geo_point
+
+    - name: hostname
+      type: string
+      index: not_analyzed
+      description: >
+        FQDN of the entity generating the original payload
+        This field is a best effort attempt to derive this context; sometimes the entity generating it knows it; other times that entity has a restricted namespace itself, and the collector or normalizer knows that
+
+    - name: ipaddr4
+      type: ip
+      description: >
+        IP address v4 of the source server, can be an array.
+      fields:
+        - name: raw
+          ignore_above: 256
+          type: string
+          index: not_analyzed
+      norms:
+        enabled: false
+
+    - name: ipaddr6
+      type: string
+      index: not_analyzed
+      description: >
+        IP address v6 of the source server(if available).
+
+    - name: level
+      type: string
+      index: not_analyzed
+      example: info
+      description: >
```

Comment:
enumerate the baseline from syslog, any other level will be attempted to be mapped in those values by the normalizer based on semantic similarity.
or a new level will be added